test/new_relic/telemetry_sdk/harvester_test.rb

```diff
@@ -10,34 +10,47 @@ module NewRelic
   module TelemetrySdk
     class HarvesterTest < Minitest::Test
 
+      def teardown
```

Comment:
I added a configurator rest to a teardown method (as well as explicitly setting the harvester via config on a number of tests) because I noticed that Harvester tests would intermittently print logs to STDOUT. I assumed some state about the logger was being leaked, though it didn't actually cause any test failures.